test/e2e/installplan_e2e_test.go

```diff
@@ -2315,8 +2295,8 @@ var _ = Describe("Install Plan", func() {
 
 		crdPlural := genName("ins")
 		crdName := crdPlural + ".cluster.com"
-		var min float64 = 2.1
```

Comment:
Doesn't this change remove the important part of the test? Verifying that we don't hit the float -> int conversion issue?

Reply:
When we had 2 as the value we hit the float -> int conversion issue when we were upgrading all CRDs to v1. Changing to 2.1 removed us hitting this issue with v1 CRDs.

Now we are back to handling v1beta1 CRDs I moved the value back to where it was previously. Should we have another test, with real floats, for this case?